Changelog.md

```diff
@@ -20,6 +20,7 @@ Compiler Features:
 
 
 Bugfixes:
+ * Code Generator: Treat dynamically encoded but statically sized arrays and structs properly.
```

Comment:
Maybe mention 'calldata' somewhere, or does this also apply to memory or non-external functions?